sklearn/tree/export.py

```diff
@@ -373,8 +375,9 @@ def recurse(tree, node_id, criterion, parent=None, depth=0):
     return_string = False
     try:
         if out_file == 'tree.dot':
-            warnings.warn("out_file can be set to None from 0.18, "
-                          "this will be the default in 0.20.", DeprecationWarning)
+            warnings.warn("out_file can be set to None starting from 0.18. "
```

Comment:
I considered this. I decided the chances that someone was explicitly setting `out_file=tree.dot` were fairly slim. Lazy of me.

I'd usually use: `SENTINEL=object()` then `out_file=SENTINEL`.